community/kernel/src/test/java/org/neo4j/kernel/impl/store/kvstore/ConcurrentMapStateTest.java

```diff
@@ -133,52 +132,18 @@ public void shouldCreateAnUpdaterForMultipleVersionUpdatesNotInOrder() throws Ex
     }
 
     @Test
-    public void shouldFailIfApplyingAVersionUpdateWithAVersionLowerOrEqualToTheInitialVersion() throws Exception
+    public void shouldUseEmptyUpdaterOnVersionLowerOrEqualToTheInitialVersion() throws Exception
```

Comment:
yep that we have talk on top of PR, but for me it still sound like creating empty updater is still better in that case comparing to throwing exception